monai/transforms/compose.py

```diff
@@ -41,7 +41,7 @@ class Transform(ABC):
     """
 
     @abstractmethod
-    def __call__(self, data, *args, **kwargs):
+    def __call__(self, data: Any):
```

Comment:
Removing the `*args, **kargs` caused a new alert in our LGTM CI test:
https://lgtm.com/projects/g/Project-MONAI/MONAI/rev/pr-5ee820bbfb735b2e410d9d366ceb3581ff7ebc3e
You can also open the LGTM test details page to check it.
Because it's against the "Liskov Substitution Principle".
Thanks.